physical/mysql/mysql.go

```diff
@@ -620,7 +620,7 @@ func (i *MySQLLock) becomeLeader() error {
 func (i *MySQLLock) Lock() error {
 	defer metrics.MeasureSince([]string{"mysql", "get_lock"}, time.Now())
 
-	rows, err := i.in.Query("SELECT GET_LOCK(?, -1), IS_USED_LOCK(?)", i.key, i.key)
+	rows, err := i.in.Query("SELECT GET_LOCK(?, 4294967295), IS_USED_LOCK(?)", i.key, i.key)
```

Comment:
How was this number chosen?

Reply:
This number is the decimal representation of maximal integer with 64bit length. In hexadecimal this number should be 0xffffffff

Reply:
Can we instead use `math.MaxUint32` from the https://golang.org/pkg/math/ package and add the comment a comment above that about the `-1` not being compatible with different MySQL flavours?

Reply:
@michaeljs1990 Yes, we can use this constant. I tested this, committed changes and comments.

Reply:
Same problem faced. Can you help with steps on how to apply this fix on a running mysql deployment?

Reply:
You shouldn't need to make any changes to mysql for this fix. What is the issue you are running into?